frontend/online-shop/src/components/header/header.component.tsx

```diff
@@ -0,0 +1,28 @@
+import { Typography } from "@mui/material";
+import { Link } from "react-router-dom";
+import { AiOutlineShoppingCart } from "react-icons/ai";
+import { CartLink, HeaderComponent, OptionLink } from "./header.style";
+const Header = () => {
```

Comment:
You need an empty line between the imports and the following piece of code so it's easier to read